src/test/java/core/basesyntax/RegistationServiceTest.java

```diff
@@ -0,0 +1,204 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.exception.ValidatorException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistationServiceTest {
+    private static RegistrationService registerServiceTest;
+    private static List<User> users = initCorrectUsersList();
+
+    private List<User> initUsersList() {
+        List<User> usersList = new ArrayList<>();
+        usersList.add(new User("BobA", "4385482", 43));
+        usersList.add(new User(null, "655234322", 21));
+        usersList.add(new User("AlexFreedom", "543234113", null));
+        usersList.add(new User("NickKarlosn", null, 43));
+        usersList.add(new User("YuliaMikovskay", "3244", 56));
+        usersList.add(new User("AlenTurke", "3243234", 17));
+        usersList.add(new User("", "3243234", 43));
+        usersList.add(new User("LizaVavilova", "", 24));
+        usersList.add(new User("AndrewSmith", "43543535", -10));
+        usersList.add(new User("4LizaValencia", "43243234", 54));
+
+        return usersList;
+    }
+
+    private static List<User> initCorrectUsersList() {
+        List<User> correctList = new ArrayList<>();
+        correctList.add(new User("FillCurrent", "12345678", 23));
+        correctList.add(new User("JhonMelkavich", "987654321", 34));
+        correctList.add(new User("AlenaBekovskay", "123987654", 18));
+        correctList.add(new User("RitaVonalich", "999384633", 37));
+
+        return correctList;
+    }
+
+    @BeforeAll
+    static void createService() {
+        registerServiceTest = new RegistrationServiceImpl();
+        for (User user : users) {
+            registerServiceTest.register(user);
+        }
+    }
+
+    @Test
+    void register_isNullUser_NotOk() {
+        User nullUser = null;
+        assertThrows(NullPointerException.class, () -> {
+            registerServiceTest.register(nullUser);
+        });
+    }
+
+    @Test

```

Comment:
```suggestion
    @Test(expected = ValidationException.class)
```
use this construct https://www.baeldung.com/junit-assert-exception#junit-4
everywhere

Reply:
dont understand what i should to do